index.js

```diff
@@ -331,20 +326,11 @@ function processResponseBody (options, response, body) {
     body = body.toString(options.realEncoding);
     // The resolveWithFullResponse option will resolve with the response
     // object. This changes the response.body so it is as expected.
-    response.body = body;
-
-    if (response.isCloudflare) {
```

Comment:
Why was this removed?

Reply:
TBH, it never made any sense to me but with recent changes it became clear that it wasn't necessary.